src/base/NekRSProblem.C

```diff
@@ -218,6 +223,23 @@ NekRSProblem::initialSetup()
       }
   }
 
+  if (boundary && nekrs::hasMovingMesh() && nekrs::hasElasticitySolver())
```

Comment:
I think that `nekrs::hasMovingMesh()` can be removed here, because `hasMovingMesh()` will be true as long as either `solver = user` or `solver = elasticity`

Reply:
Done, thanks!